Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/QuantitySelector.kt

```diff
@@ -0,0 +1,77 @@
+package com.example.jetsnack.ui.components
+
+import androidx.compose.foundation.Text
+import androidx.compose.foundation.layout.ChainStyle
+import androidx.compose.foundation.layout.ConstraintLayout
+import androidx.compose.foundation.layout.preferredWidthIn
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.AddCircleOutline
+import androidx.compose.material.icons.outlined.RemoveCircleOutline
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.ui.tooling.preview.Preview
+import com.example.jetsnack.R
+import com.example.jetsnack.ui.theme.JetsnackTheme
+
+@Composable
+fun QuantitySelector(
+    count: Int,
+    updateCount: (Int) -> Unit,
+    modifier: Modifier = Modifier
+) {
+    ConstraintLayout(modifier = modifier) {
+        val (qty, minus, quantity, plus) = createRefs()
+        createHorizontalChain(qty, minus, quantity, plus, chainStyle = ChainStyle.Packed)
+        Text(
```

Comment:
Wrap this in a `ProvideEmpashis(AmbientEmphasisLevels.XXX) { ... }`

Reply:
How does this work exactly? The documentation says:
- `AmbientEmphasisLevels.current.high` -  Emphasis used to express high emphasis, such as for selected text fields.
- `AmbientEmphasisLevels.current.medium` - Emphasis used to express medium emphasis, such as for placeholder text in a text field.
- `AmbientEmphasisLevels.current.disabled` - Emphasis used to express disabled state, such as for a disabled button.

None of these seem to match.

Reply:
Most body text is high, which is what should be used here.